recipes/server_debian.rb

```diff
@@ -30,3 +30,10 @@
   supports :restart => true, :status => true, :reload => true
   action [:enable, :start]
 end
+
+unless ::File.directory?('/etc/postgresql/' + node['postgresql']['version'] + '/main')
```

Comment:
So you know, this can be removed in favor of using a not_if for the execute resource, see https://docs.chef.io/resource_common.html#not-if-examples for some examples.